example/index.tsx

```diff
@@ -14,15 +16,22 @@ const config = new DdSdkReactNativeConfiguration(
     ENVIRONMENT,
     APPLICATION_ID,
     true,
+    true,
     true
 )
+config.nativeCrashReportEnabled = true
+config.sampleRate = 100
 
 DdSdkReactNative.initialize(config).then(() => {
 
     DdLogs.info('The RN Sdk was properly initialized', {
         foo: 42,
         bar: 'xyz',
     })
+
+    DdSdk.setUser({id: "1337", name: "Xavier", email: "xg@example.com", type: "premium"})
+
```

Comment:
it is quite confusing to call these methods from the `DdSdk` but the initialization from the `DdSdkReactnative`

Reply:
For me it seems overkill to artificially create a method in DdSdkReactnative that would add no value other than transferring the call to DdSdk. 
Besides for people manually instrumenting their code, they would not use the DdSdkReactnative class.